collector/collector.go

```diff
@@ -167,19 +167,22 @@ func (p PostgresCollector) Describe(ch chan<- *prometheus.Desc) {
 func (p PostgresCollector) Collect(ch chan<- prometheus.Metric) {
 	ctx := context.TODO()
 
+	// copy the instance so that concurrent scrapes have independent instances
```

Comment:
```suggestion
	// Copy the instance so that concurrent scrapes have independent instances.
```

Reply:
Just to obey the silent formatting convention :)

Reply:
it will lead to conflict with upstream :) I cherry-picked changes from that side